src/base/compute/owl_computation_shape.ml

```diff
@@ -55,11 +55,12 @@ module Make (Type : Owl_computation_type_sig.Sig) = struct
 
   let _infer_shape_07 input_shapes axis =
     let s0 = Array.map (fun s -> s.(0)) input_shapes in
-    if Array.exists
-         (function
-           | Some _ -> false
-           | None   -> true)
-         s0
+    if
+      Array.exists
+        (function
+          | Some _ -> false
+          | None   -> true)
```

Comment:
again, personally i would find this clearer if the closing `)` were on a separate line, indented to line up with the opening `(`

Reply:
`indicate-multiline-delimiters=closing-on-separate-line` should give that behavior as of ocamlformat 0.16.0